frontend/src/routes/CriticalQuestions/index.js

```diff
@@ -151,15 +158,15 @@ function CriticalQuestions(props) {
         nextPage = "/dashboard";
     }
     else if (tested === "positive") {
-        nextPage = "/symptoms"
+        nextPage = "/feeling"
     }
     else {
-        nextPage = "/symptoms"
+        nextPage = "/feeling"
```

Comment:
We should also use the `paths.feeling` constant here, to avoid any possible typos 🙏